app.test.js

```diff
@@ -18,29 +18,67 @@ beforeEach(() => {
 });
 
 app.use((err, req, res, next) => {
-    if (err.status && err.msg) {
-      res.status(err.status).send({ msg: err.msg });
-    } else {
-      next(err);
-    }
-  });
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+});
 
-app.use((err,req,res,next)=>{
-    res.status(500).send({msg :"Code 500 - server error!"})
-})
+app.use((err, req, res, next) => {
+  res.status(500).send({ msg: "Code 500 - server error!" });
+});
 
 describe("/api/topics", () => {
-    test.only("GET request - status 200 responds with all the topics sorted by age", () => {
-      return request(app)
-        .get("/api/topics")
-        .expect(200)
-        .then((response) => {
-          const topics = response.body.topics;
-          expect(topics.length).toEqual(topicData.length);
-          topics.forEach((topic) => {
-            expect(typeof topic.slug).toEqual("string");
-            expect(typeof topic.description).toEqual("string");
-          });
+  test("GET request - status 200 responds with all the topics sorted by age", () => {
```

Comment:
Careful with your test descriptions. Are topics sorted by age? Why would we need that? And you aren't asserting that anywhere in your test. Make sure your descriptions are matching what you are actually testing.